heterogeneity/Documentation_JV_Summer2022

```diff
@@ -0,0 +1,194 @@
+We were only able to partially replicate the results for California. 
+Most of the work consisted on running the python scripts in the correct order and troubleshoot the possible mistakes and missing information we found on the pythion scripts. 
+Most of the work was made in Python using the Spyder app. 
+
+The topics I worked are the following:
+1)	Creating a working environment in Python 
+2)	Order of the python scripts 
+3)	Input and output of every python script  
+4)	Running time consuming scripts in the SU cluster.
+5)  Where my work ended.
+I will write a description for each with the proper procedure.
+
+1)	Creating a working environment in Python 
+
+The following steps create the environment called ‘Test_Jorge’, any other name would do. 
+This environment is created only in the remote computer I was using. Professor Buzard is looking into creating the environment for every computer.
+
+1) In Anaconda Command Prompt typed:
+* conda create --name Test_Jorge python=3.7
+* conda activate Test_Jorge
+
+2) Install the packages in Anaconda Command Prompt:
+* conda install -c esri arcgis
+* conda install -c conda-forge mkl-service
+* conda install -c conda-forge plotnine
+* conda install -c conda-forge geopandas
+* conda install -c conda-forge geopy
+* conda install -c conda-forge opencv-python
+* conda install -c conda-forge pytesseract
+* conda install -c conda-forge pyap
+* conda install -c conda-forge tabulate
+* conda install -c conda-forge openpyxls (if do not work try "pip install openpyxls")
+
+Note: Every time you need to install a new package, do it in Anaconda Command Prompt.
+
+3) Go to the "Anaconda Navigator" on the Start Menu:
+
+* On Anaconda Navigator go to environment and select "Test_Jorge" 
+* Once "Test_Jorge" is installed, go to Home and install "Spyder". If "Spyder" is already installed, then click "Launch".
+
+4) In Spyder, open the script you need. 
+
+Note: If Spyder gets stuck on "connecting to kernel", then close Spyder and do the following:
+4.1) Go to "Environments" in the Anaconda Navigator.
+4.2) Search for pyzmq
+4.3) Click on the green tick box and select version 16.0.4
+4.4) Click on "Apply"
+4.5) Reopen Spyder
+
+Now you have set up your environment. Go to Spyder and run the scripts. 
+
+## Order of the Python Scripts. 
+
+## Diagram for Clarification
+
+2)	Order of the python scripts
+
+First, all the python scripts are located in:
+G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\ramosRivera\T-Burk\Python Scripts
+
+To help understand what we are doing is good to start with the intuition of the procedure. 
+First, we are digitalizing the addresses of the labs. 
+Second, we are Geocoding those address to be able to locate them in a map using ArcMap. 
+Third, we are creating the clusters by running simulations. 
+Fourth, we are calculating the agglomerations effects. 
+
+Next, the python scripts should be run in the following order:
```

Comment:
This "Next" is confusing with the First, Second, etc. list above it. It would be clearer to replace it with something like, "In order to carry out the above four tasks,"